src/solve/client/client.ts

```diff
@@ -44,4 +130,711 @@ export class SkipClient {
       (chain) => !this.ignoreChains.includes(chain.chain_id)
     );
   }
+
+  async executeRoute(
```

Comment:
I think we should separate out the executeRoute logic into a second PR from the local testnet code.